tools/exports.js

```diff
@@ -17,3 +17,4 @@ exports["string_template"] = string_template;
 exports["tokenizer"] = tokenizer;
 exports["is_identifier"] = is_identifier;
 exports["SymbolDef"] = SymbolDef;
+exports["EXPECT_DIRECTIVE"] = EXPECT_DIRECTIVE;
```

Comment:
Actually no. A way to have private access would be desireable. I don't like to test this as 2 disconnected pieces of code.